brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/Alien4CloudFacade.java

```diff
@@ -419,15 +437,40 @@ public Object resolveProperty(String nodeId, Alien4CloudApplication toscaApplica
 
     @Override
     public Map<String, Object> getPropertiesAndTypeValues(String nodeId, Alien4CloudApplication toscaApplication, String requirementId, String computeName) {
-        Optional<RelationshipTemplate> optionalRelationshipTemplate = findRelationshipRequirement(nodeId, toscaApplication, requirementId);
-        if (optionalRelationshipTemplate.isPresent()) {
-            RelationshipTemplate relationshipTemplate = optionalRelationshipTemplate.get();
+
+        Map<String, Object> result = MutableMap.of();
+        Set<RelationshipTemplate> relationshipTemplates =
+                findRelationshipsRequirement(nodeId, toscaApplication, requirementId);
+
+        for (RelationshipTemplate relationshipTemplate : relationshipTemplates) {
             if (relationshipTemplate.getType().equals("brooklyn.relationships.Configure")) {
                 Map<String, Object> relationProperties = getRelationProperties(nodeId, computeName, toscaApplication, relationshipTemplate);
-                return getPropertiesAndTypedValues(relationshipTemplate, relationProperties, computeName);
+                result = joinPropertiesAndValueTypes(result,
+                        getPropertiesAndTypedValues(relationshipTemplate, relationProperties, computeName));
+            }
+        }
+        return result;
+    }
+
+    @SuppressWarnings("unchecked")
+    private Map<String, Object> joinPropertiesAndValueTypes(Map<String, Object> properties,
+                                                            Map<String, Object> newProperties) {
+        for (String newPropertyId : newProperties.keySet()) {
+            Object newPropertyValue = newProperties.get(newPropertyId);
+            if (!properties.containsKey(newPropertyId)) {
+                properties.put(newPropertyId, newPropertyValue);
+            } else {
+                Object oldPropertyValue = properties.get(newPropertyId);
+                if ((oldPropertyValue instanceof Map)
+                        && (newPropertyValue instanceof Map)) {
+                    ((Map) oldPropertyValue).putAll((Map) newPropertyValue);
+                } else if ((oldPropertyValue instanceof List)
+                        && (newPropertyValue instanceof List)) {
+                    ((List) oldPropertyValue).addAll((List) newPropertyValue);
+                }
```

Comment:
Should log a message to the debug log if no action is to be taken with the new property.